calibration/DOSTAM/CGINS-DOSTAM-00891__20200614.csv

```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+891,CC_calibration_date,1592092800000,sensor calibration date (milliseconds since 1970-01-01)
```

Comment:
looks correct, but no calibration file available on vault/alfresco to verify cal date

Reply:
confirmed - looks good